lib/sound/track.cpp

```diff
@@ -27,7 +27,7 @@
 //*
 //
 // defines
-#define MAX_TRACKS	( 600 )
+#define MAX_TRACKS	( 65536 )
```

Comment:
This seems like quite the jump - how many more tracks does your mod actually add?

Reply:
I think I counted about 900 unique files, I'm figuring that jumping it to this number will prevent crashing into it for a good long while into the future

The reason for so many files is that the mod's meant to add unique voice acting to droids, and this game has a massive variety of them